packages/dds/tree/src/feature-libraries/modular-schema/fieldKindWithEditor.ts

```diff
@@ -1,108 +0,0 @@
-/*!
- * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
- * Licensed under the MIT License.
- */
-
-import { assert } from "@fluidframework/core-utils/internal";
-
-import type {
-	FieldKindIdentifier,
-	TreeFieldStoredSchema,
-	TreeStoredSchema,
-	TreeTypeSet,
-	Multiplicity,
-} from "../../core/index.js";
-
-import type { FieldChangeHandler, FieldEditor } from "./fieldChangeHandler.js";
-import { FlexFieldKind, type FullSchemaPolicy } from "./fieldKind.js";
-import { isNeverField } from "./isNeverTree.js";
-
-// TODO: stronger typing
-/* eslint-disable @typescript-eslint/no-explicit-any */
-
-/**
- * Functionality for FieldKinds that is stable,
- * meaning that it can not change in any measurable way without providing a new identifier.
- *
- * It is assumed that this information and policy is available on all clients interacting with a document
- * using the identifier.
- *
- * This must contain enough information to process remote edits to this FieldKind consistently with all clients.
- * All behavior must be deterministic, and not change across versions of the app/library.
- *
- * These policies include the data encoding, change encoding, change rebase and change application.
```

Comment:
These docs are the same as the ones on FlexFieldKind, so this change deduplicates them.

The rest of this class is just moved inline into FlexFieldKind

Reply:
Yeah, what's what we get for changing who sets the vision for this stuff three times.